metricbeat/module/system/uptime/metricset_test.go

```diff
@@ -0,0 +1,28 @@
+// +build darwin linux openbsd windows
```

Comment:
@andrewkroh was there any reason to exclude freebsd here? sigar has the [implementation for this](https://github.com/elastic/gosigar/blob/master/sigar_freebsd.go#L38). // cc @kaiyan-sheng @ruflin

Reply:
I can’t think of any reason. It probably should be tagged as `freebsd,cgo`.